### ISSUE TITLE
Terminal Picture Viewer

### ISSUE THREAD

**Issue by habitualdev** [antonmedv/llama#14]
Wanted to know your thoughts on the addition of functionality to "print" a picture to the terminal. Got it working with a modified version of https://github.com/ichinaski/pxl, and appears to work in the terminals I've tested it in. 

On one hand, I've been getting actual use out of it (remote viewing of scraped webpages/assets and remote rapid forensics), but on the other understand it could be seen as feature creep. I have a working version on a seperate branch on my fork. In its current implementation it does add another key ("q" for quit).

**Comment by antonmedv** [comment]
I’m thinking of adding support for different file extension/program mapping via ~/.llama config in future. 

But waiting on upstream dependency- bubbletea, to support sub process running.

**Comment by roaldi** [comment]
Sounds good. For jpeg/png I actually just directly integrated the pxl code after I refactored it to be an importable module. And I can probably add support for more picture formats as well

**Comment by antonmedv** [comment]
Duplicate of #26
